src/styles/pages/_details.scss

```diff
@@ -0,0 +1,61 @@
+main {
```

Comment:
- [ ] [OPTIONAL] There is a little space between navigation and the `Showing the 30 Most Active Stocks` part. Let's also remove it 🙃

Screenshot
![image](https://user-images.githubusercontent.com/69011963/157240625-a9dee92e-7ea2-44bf-aea9-fd7c90b19c3f.png)

Reply:
@carloshs1994 You are right, problem is with my simulator not your mistake sorry 👍